src/c_api.rs

```diff
@@ -833,8 +829,7 @@ pub unsafe extern "C" fn blazesym_find_address_regex(
 #[no_mangle]
 pub unsafe extern "C" fn blazesym_syms_free(syms: *const blazesym_sym_info) {
     if syms.is_null() {
-        #[cfg(debug_assertions)]
-        eprintln!("blazesym_sym_info_free(null)");
+        warn!("blazesym_sym_info_free(null)");
```

Comment:
on C API side it's customary for free-like functions to accept NULL and do nothing at all, it's not really considered an error. Let's drop warnings for such cases?

Reply:
Sure, will adjust.